package.xml

```diff
@@ -15,21 +18,24 @@
   <author>Brian Gerkey</author>
   <author>Dirk Thomas</author>
 
+  <depend condition="$ROS_PYTHON_VERSION == 2">python-argparse</depend>
+  <depend condition="$ROS_PYTHON_VERSION == 2" version_gt="0.4.3">python-catkin-pkg</depend>
+  <depend condition="$ROS_PYTHON_VERSION == 3" version_gt="0.4.3">python3-catkin-pkg</depend>
+  <depend condition="$ROS_PYTHON_VERSION == 2">python-empy</depend>
+  <depend condition="$ROS_PYTHON_VERSION == 3">python3-empy</depend>
```

Comment:
Is `python3-empy` needed at runtime? It looks like it was listed as `<build_depend>` + `<build_export_depend>` before.

Reply:
`build_export_depend` is technically a run dependency. The only difference is that `build_export_depend` implies that it is used on the architecture the code is built on whereas `exec_depend` implies the target architecture. For Python packages (without C extensions) that distinction is irrelevant. Therefore I chose to simplify the dependency declaration.